apigw-custom-domain/serverless.yml

```diff
@@ -2,9 +2,9 @@ service: sls-apigw-domain
 
 provider:
   name: aws
-  runtime: nodejs16.x
+  runtime: nodejs18.x
```

Comment:
which paywall is node20.x locked behind, serverless framework?

Reply:
Yes, requires upgrade to Serverless v4 which requires us to subscribe to a plan